.github/workflows/ci.yml

```diff
@@ -40,6 +40,9 @@ jobs:
       - run: pnpm build:ci
         env:
           NODE_OPTIONS: --max_old_space_size=4096
+
+      - run: pnpm dlx pkg-pr-new publish './packages/*' --template './examples/*' --compact
```

Comment:
It looks like this should run in CircleCI. We test the build flow on different OS here.

Reply:
```suggestion
      - run: pnpm dlx pkg-pr-new publish './packages/*' --template './examples/*' --compact --comment=off
```

Reply:
@oliviertassinari Can you install the app on this repo? I don't seem to have write access.

Reply:
I believe it's already installed.

Reply:
There would be only one comment by pkg.pr.new and that will be updated again & again! But anyway, I'll sort this one out.

Reply:
> It looks like this should run in CircleCI

Oh, I see, pkg.pr.new is not supported on CircleCI, it needs to be run in github workflows only.